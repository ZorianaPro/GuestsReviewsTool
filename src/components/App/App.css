@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,500;0,900;');

:root {
  --colorWhite: #fff;
  --colorBlue: #1576DB;
  --colorBlack: #000;
  --colorLightGray: #E5E5E5;
  --colorGray: #646464;

  --fontFamilyRoboto: 'Roboto', sans-serif;
  --lineHeight: 1.63;
  --fontBold: 900;

  --borderRadius: 0.75em;

  --fontSize4: .125em;
  --fontSize8: .25em;
  --fontSize12: .75em;
  --fontSize14: .875em;
  --fontSize16: 1em;
  --fontSize18: 1.125em;
  --fontSize20: 1.25em;
  --fontSize24: 1.5em;
  --fontSize26: 1.62em;
  --fontSize30: 1.875em;
  --fontSize36: 2.25em;
  --fontSize48: 4em;
}

.App {
  position: relative;
  width: 100%;
  height: 100%;
  min-height: 100vh;
  margin: 0 auto;
  align-content: flex-start;
  display: flex;
  flex-direction: column;
}

.App b {
  font-weight: var(--fontBold);
}

h1 {
  font-size: var(--fontSize26);
}

h2 {
  font-size: var(--fontSize24);
  margin: 0
}

h3 {
  font-size: var(--fontSize20);
  margin: 0
}

h4 {
  font-size: var(--fontSize16);
  margin: 0
}

h5 {
  font-size: var(--fontSize14);
  color: var(--colorGray);
  font-weight: normal;
  margin: 0;
}