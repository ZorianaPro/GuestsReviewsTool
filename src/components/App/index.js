import React from 'react';
import { Switch, Route, Redirect } from 'react-router-dom';
import GuestReviews from '../GuestReviews';

const App = () => (
  <div className="App">
    <Switch>
      <Route path="/reviews/">
        <GuestReviews/>
      </Route>
      <Route path="*">
        <Redirect to="/reviews/" />
      </Route>
    </Switch>
  </div>
);

export default App;
