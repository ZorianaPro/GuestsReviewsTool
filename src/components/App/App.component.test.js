import React from 'react';
import { MemoryRouter as Router } from 'react-router-dom';
import { render, shallow } from 'enzyme';
import App from './';

describe('App', () => {
  it('renders without crashing', () => {
    expect(
      shallow.bind(
        shallow,
        <Router>
          <App />
        </Router>
      )
    ).not
      .toThrow();
  });

  it('renders as expected', () => {
    expect(
      render(
        <Router>
          <App />
        </Router>
      )
    ).toMatchSnapshot();
  });

  describe('invalid path', () => {
    beforeEach(() => {
      const app = shallow(
        <Router initialEntries={[ '/random' ]}>
          <App/>
        </Router>
      );
      it('should redirect to GuestReviews page', () => {
        expect(app.exists('.GuestReviews'));
      });
    });
  });
});
