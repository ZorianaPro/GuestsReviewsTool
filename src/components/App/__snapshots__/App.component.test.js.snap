// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders as expected 1`] = `
<div
  class="App"
>
  <svg
    class="SVGSpriteSheet"
    id="SVGSpriteSheet"
  >
    <symbol
      class="AirbnbSymbol"
      id="AirbnbSymbol"
      version="1.1"
      viewBox="0 0 88 27"
      xlink="http://www.w3.org/1999/xlink"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M46.38 6.784c0 .973-.798 1.757-1.788 1.757s-1.787-.784-1.787-1.757c0-.973.77-1.757 1.787-1.757 1.017.027 1.787.81 1.787 1.757zm-7.368 3.54v.433s-.853-1.081-2.667-1.081c-2.996 0-5.333 2.243-5.333 5.351 0 3.081 2.309 5.351 5.333 5.351 1.842 0 2.667-1.108 2.667-1.108v.46c0 .216.165.378.384.378h2.227V9.946h-2.227a.39.39 0 00-.384.378zm0 6.514c-.413.594-1.237 1.108-2.227 1.108-1.76 0-3.107-1.081-3.107-2.919s1.347-2.919 3.107-2.919c.962 0 1.842.54 2.227 1.108v3.622zm4.26-6.892h2.64v10.162h-2.64V9.946zm39.422-.27c-1.814 0-2.666 1.08-2.666 1.08V5.055h-2.64v15.054h2.227A.39.39 0 0080 19.73v-.46s.852 1.108 2.667 1.108c2.996 0 5.333-2.27 5.333-5.351s-2.337-5.351-5.306-5.351zm-.44 8.243c-1.017 0-1.814-.514-2.226-1.108v-3.622c.412-.54 1.292-1.108 2.226-1.108 1.76 0 3.107 1.081 3.107 2.919s-1.347 2.919-3.107 2.919zm-6.24-3.838v6.054h-2.639v-5.757c0-1.675-.55-2.351-2.034-2.351-.798 0-1.622.405-2.145 1v7.081h-2.639V9.946h2.09c.22 0 .384.19.384.378v.433c.77-.784 1.787-1.081 2.804-1.081 1.155 0 2.117.324 2.887.973.935.756 1.292 1.73 1.292 3.432zM60.152 9.676c-1.814 0-2.667 1.08-2.667 1.08V5.055h-2.639v15.054h2.227a.39.39 0 00.385-.378v-.46s.852 1.108 2.666 1.108c2.997 0 5.334-2.27 5.334-5.351.027-3.081-2.31-5.351-5.306-5.351zm-.44 8.243c-1.017 0-1.814-.514-2.227-1.108v-3.622c.413-.54 1.292-1.108 2.227-1.108 1.76 0 3.107 1.081 3.107 2.919s-1.348 2.919-3.107 2.919zm-7.148-8.243c.798 0 1.21.135 1.21.135v2.405s-2.2-.73-3.574.811v7.108h-2.639V9.945h2.227c.22 0 .385.19.385.38v.432c.495-.568 1.567-1.081 2.391-1.081zm-27.408 9.513c-.137-.324-.275-.675-.412-.973-.22-.486-.44-.946-.632-1.378l-.028-.027A280.121 280.121 0 0018.01 4.595l-.083-.163c-.22-.405-.44-.837-.66-1.27-.274-.486-.55-1-.99-1.486A4.468 4.468 0 0012.787 0c-1.375 0-2.612.595-3.52 1.622-.412.486-.714 1-.989 1.486-.22.433-.44.865-.66 1.27l-.082.163c-2.117 4.054-4.179 8.162-6.076 12.216l-.027.054c-.193.432-.413.892-.632 1.378a17.63 17.63 0 00-.413.973c-.357 1-.467 1.946-.33 2.92a5.762 5.762 0 003.574 4.486A5.73 5.73 0 005.858 27a7.348 7.348 0 003.464-.973c1.127-.622 2.199-1.513 3.408-2.81 1.21 1.297 2.31 2.188 3.41 2.81a7.348 7.348 0 003.463.973c.77 0 1.54-.135 2.227-.432a5.75 5.75 0 003.574-4.487c.22-.946.11-1.892-.248-2.892zm-12.398 1.406c-1.485-1.838-2.447-3.568-2.777-5.027-.137-.622-.165-1.163-.082-1.65.055-.432.22-.81.44-1.134.522-.73 1.402-1.19 2.419-1.19 1.017 0 1.924.433 2.42 1.19.22.324.384.702.439 1.135.082.486.055 1.054-.082 1.649-.33 1.432-1.293 3.162-2.777 5.027zm10.969 1.27c-.193 1.405-1.155 2.621-2.502 3.162-.66.27-1.374.351-2.09.27a5.47 5.47 0 01-2.088-.702c-.99-.54-1.98-1.379-3.134-2.622 1.814-2.19 2.914-4.19 3.326-5.973.192-.838.22-1.595.137-2.297a4.343 4.343 0 00-.742-1.838c-.852-1.216-2.282-1.92-3.876-1.92-1.594 0-3.024.73-3.876 1.92a4.343 4.343 0 00-.742 1.838A5.52 5.52 0 008.277 16c.412 1.784 1.54 3.81 3.326 6-1.127 1.243-2.144 2.081-3.134 2.622a5.47 5.47 0 01-2.089.702 4.467 4.467 0 01-2.09-.27c-1.346-.54-2.308-1.757-2.5-3.162a4.404 4.404 0 01.247-2.108c.082-.27.22-.54.357-.865.192-.433.412-.892.632-1.351l.028-.054A293.393 293.393 0 019.102 5.378l.082-.162c.22-.405.44-.838.66-1.243.22-.432.467-.838.77-1.19.577-.648 1.347-1 2.199-1 .852 0 1.622.352 2.2 1 .302.352.549.758.769 1.19.22.405.44.838.66 1.243l.082.162a355.977 355.977 0 016.02 12.163v.027c.22.432.413.918.633 1.35.137.325.275.596.357.866.22.702.303 1.378.193 2.08z"
        fill="#FF5A5F"
        fill-rule="evenodd"
      />
    </symbol>
    <symbol
      class="BookingcomSymbol"
      id="BookingcomSymbol"
      viewBox="0 0 154 26"
      xlink="http://www.w3.org/1999/xlink"
      xmlns="http://www.w3.org/2000/svg"
    >
      <g
        fill="none"
        fill-rule="nonzero"
      >
        <path
          d="M60.25 2.41c0-1.288 1.039-2.33 2.318-2.33a2.328 2.328 0 010 4.658 2.324 2.324 0 01-2.319-2.329"
          fill="#273B7D"
        />
        <path
          d="M96.904 17.902a2.325 2.325 0 012.318-2.331 2.33 2.33 0 010 4.66 2.325 2.325 0 01-2.318-2.33"
          fill="#499FDD"
        />
        <path
          d="M22.122 16.895c-2.001 0-3.391-1.59-3.391-3.86 0-2.27 1.39-3.858 3.39-3.858 2.013 0 3.419 1.588 3.419 3.858 0 2.307-1.377 3.86-3.418 3.86zm0-11.11c-4.233 0-7.305 3.05-7.305 7.25s3.072 7.25 7.305 7.25c4.248 0 7.333-3.05 7.333-7.25s-3.085-7.25-7.333-7.25zM55.75 13.398a3.456 3.456 0 00-.554-.78l-.128-.136.134-.13c.195-.204.393-.447.585-.728l3.735-5.553h-4.534l-2.807 4.344c-.159.232-.48.35-.96.35h-.64V2.553c0-1.643-1.023-1.866-2.128-1.866H46.56l.004 19.384h4.018v-5.814h.376c.458 0 .77.053.913.301l2.217 4.184c.619 1.135 1.236 1.33 2.397 1.33h3.079l-2.293-3.794-1.521-2.88M75.256 5.754c-2.043 0-3.348.908-4.078 1.676l-.244.246-.086-.335c-.215-.823-.939-1.275-2.028-1.275h-1.8l.01 14h3.992v-6.452c0-.631.082-1.179.248-1.678.445-1.512 1.684-2.452 3.232-2.452 1.246 0 1.733.657 1.733 2.356v6.098c0 1.45.67 2.128 2.122 2.128h1.899l-.007-8.904c0-3.538-1.725-5.408-4.991-5.408M62.621 6.072H60.73l.013 10.825v3.173h2.018l.071.002.94-.002h.937v-.006h.006l.008-11.867c0-1.431-.684-2.125-2.1-2.125M37.797 16.895c-2 0-3.394-1.59-3.394-3.86 0-2.27 1.393-3.858 3.394-3.858 2.008 0 3.418 1.588 3.418 3.858 0 2.307-1.376 3.86-3.418 3.86zm0-11.11c-4.237 0-7.31 3.05-7.31 7.25s3.073 7.25 7.31 7.25c4.244 0 7.334-3.05 7.334-7.25s-3.09-7.25-7.334-7.25z"
          fill="#273B7D"
        />
        <path
          d="M123.34 16.895c-2 0-3.394-1.59-3.394-3.86 0-2.27 1.394-3.858 3.394-3.858 2.008 0 3.418 1.588 3.418 3.858 0 2.307-1.376 3.86-3.418 3.86zm0-11.11c-4.237 0-7.31 3.05-7.31 7.25s3.073 7.25 7.31 7.25c4.244 0 7.334-3.05 7.334-7.25s-3.09-7.25-7.334-7.25z"
          fill="#499FDD"
        />
        <path
          d="M88.664 16.279c-2.182 0-2.958-1.904-2.958-3.688 0-.786.198-3.347 2.748-3.347 1.268 0 2.957.363 2.957 3.48 0 2.938-1.495 3.555-2.747 3.555zm4.818-10.245c-.757 0-1.34.302-1.632.854l-.11.214-.185-.16c-.644-.558-1.799-1.22-3.675-1.22-3.731 0-6.244 2.803-6.244 6.974 0 4.166 2.6 7.078 6.32 7.078 1.27 0 2.274-.297 3.07-.9l.308-.23v.388c0 1.87-1.21 2.902-3.402 2.902-1.065 0-2.035-.26-2.683-.495-.847-.255-1.343-.044-1.686.806l-.317.783-.448 1.145.277.148c1.403.745 3.23 1.189 4.88 1.189 3.4 0 7.368-1.74 7.368-6.635l.014-12.84h-1.854zM7.26 16.734l-3.25-.003v-3.884c0-.83.322-1.262 1.034-1.361h2.215c1.581 0 2.603.997 2.604 2.61-.001 1.656-.997 2.637-2.604 2.638zM4.01 6.257V5.235c0-.895.379-1.32 1.21-1.375h1.662c1.426 0 2.28.853 2.28 2.28 0 1.089-.585 2.359-2.227 2.359H4.01V6.257zm7.398 3.876l-.587-.33.512-.438c.597-.513 1.596-1.665 1.596-3.657 0-3.047-2.363-5.013-6.021-5.013H2.734V.694h-.476A2.037 2.037 0 00.294 2.707V20H6.991c4.066 0 6.69-2.214 6.69-5.642 0-1.845-.848-3.423-2.274-4.226h.001z"
          fill="#273B7D"
        />
        <path
          d="M148.5 5.8a5.502 5.502 0 00-4.284 2.075l-.289.367-.227-.41c-.739-1.348-2.01-2.033-3.78-2.033-1.855 0-3.099 1.035-3.677 1.65l-.378.41-.146-.539c-.21-.775-.9-1.202-1.947-1.202h-1.677l-.016 13.946h3.81v-6.156c0-.54.068-1.073.203-1.631.364-1.488 1.364-3.088 3.044-2.928 1.037.1 1.543.9 1.543 2.448v8.267h3.838v-6.156c0-.674.063-1.179.215-1.685.309-1.42 1.353-2.876 2.974-2.876 1.174 0 1.607.665 1.607 2.45v6.228c0 1.41.629 2.039 2.037 2.039h1.794l.003-8.903c0-3.557-1.566-5.362-4.647-5.362M113.824 15.222c-.011.014-1.651 1.743-3.81 1.743-1.968 0-3.955-1.207-3.955-3.902 0-2.326 1.54-3.952 3.747-3.952.716 0 1.53.256 1.658.688l.019.073c.293.979 1.183 1.031 1.358 1.031l2.088.002V9.08c0-2.408-3.063-3.28-5.123-3.28-4.407 0-7.605 3.068-7.605 7.293 0 4.223 3.163 7.287 7.524 7.287 3.783 0 5.841-2.487 5.86-2.512l.11-.135-1.653-2.743-.218.232"
          fill="#499FDD"
        />
      </g>
    </symbol>
    <symbol
      class="HoliduSymbol"
      id="HoliduSymbol"
      viewBox="0 0 134 27"
      xlink="http://www.w3.org/1999/xlink"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M8.343 22.718c.562.243 1.205.486 1.92.715 2.09.67 4.37 1.085 6.773 1.133-.48-.249-.942-.514-1.39-.795a16.619 16.619 0 01-2.87-2.258 11.713 11.713 0 01-.711-.77 7.41 7.41 0 01-.266-.336l-.225-.315V8.73L21.66.467a1.244 1.244 0 011.761-.183l9.791 8.084c.28.23.441.571.441.93v10.41c0 .25-.078.494-.225.7a7.41 7.41 0 01-.266.336c-.197.235-.434.493-.712.769a16.619 16.619 0 01-2.87 2.258c-.446.28-.91.546-1.389.795a23.763 23.763 0 006.773-1.133 20.917 20.917 0 001.918-.714 4.137 4.137 0 01-.157-1.133c0-2.314 1.903-4.19 4.25-4.19s4.25 1.876 4.25 4.19c0 2.314-1.903 4.19-4.25 4.19a4.27 4.27 0 01-2.745-.992 23.129 23.129 0 01-2.505.96 26.178 26.178 0 01-8.54 1.25 25.903 25.903 0 01-4.572-.49 25.904 25.904 0 01-4.572.49 26.178 26.178 0 01-8.54-1.25 23.129 23.129 0 01-2.506-.96 4.27 4.27 0 01-2.746.991C1.903 25.775 0 23.9 0 21.586s1.903-4.19 4.25-4.19 4.25 1.876 4.25 4.19c0 .392-.055.772-.157 1.132zm14.271 1.3c2.143-.485 4.019-1.274 5.643-2.296a14.172 14.172 0 002.446-1.92c.188-.188.35-.362.488-.52V9.866l-8.54-7.052-8.614 7.055v9.413c.137.158.3.332.488.52.655.65 1.468 1.305 2.446 1.92 1.624 1.022 3.5 1.81 5.643 2.297zm42.057-12.92c3.326 0 5.553 2.227 5.553 5.88v8.762h-3.123v-8.448c0-2.14-1.244-3.31-3.24-3.31-2.198 0-3.673 1.34-3.673 4.11v7.648h-3.124V5.762h3.124v7.534c.955-1.455 2.43-2.197 4.483-2.197zm15.445 15.013c-2.111 0-3.933-.714-5.409-2.17-1.475-1.455-2.227-3.224-2.227-5.336s.752-3.882 2.227-5.337c1.476-1.456 3.298-2.17 5.41-2.17 2.14 0 3.933.714 5.408 2.17 1.475 1.455 2.227 3.225 2.227 5.337 0 2.112-.752 3.881-2.227 5.337-1.475 1.455-3.269 2.169-5.409 2.169zm-3.21-4.281c.867.856 1.938 1.284 3.21 1.284 1.273 0 2.343-.428 3.21-1.284.868-.856 1.302-1.94 1.302-3.225 0-1.284-.434-2.369-1.301-3.225-.868-.856-1.938-1.285-3.21-1.285-1.273 0-2.344.429-3.211 1.285-.868.856-1.302 1.94-1.302 3.225 0 1.284.434 2.369 1.302 3.225zm16.515 3.91h-3.124V4.905h3.124V25.74zm6.276-16.982c-.78.771-1.996.771-2.805 0-.781-.799-.781-1.997 0-2.768.78-.8 2.053-.8 2.834 0 .781.77.781 1.97-.029 2.768zm.174 16.982h-3.124V11.47h3.124v14.27zm14.78-12.244V5.762h3.123V25.74h-3.124v-2.055c-1.185 1.627-2.863 2.426-5.061 2.426-1.996 0-3.673-.714-5.09-2.17-1.418-1.455-2.112-3.253-2.112-5.336 0-2.112.694-3.882 2.111-5.337 1.418-1.456 3.095-2.17 5.09-2.17 2.199 0 3.877.8 5.062 2.398zm-7.839 8.391c.868.856 1.967 1.284 3.269 1.284 1.301 0 2.4-.428 3.268-1.284.868-.885 1.301-1.97 1.301-3.282 0-1.313-.433-2.398-1.301-3.254-.868-.885-1.967-1.313-3.268-1.313-1.302 0-2.401.428-3.269 1.313-.867.856-1.301 1.941-1.301 3.254s.434 2.397 1.301 3.282zm24.064-2.768v-7.65H134v14.27h-3.124v-1.826c-.954 1.456-2.43 2.198-4.483 2.198-3.326 0-5.553-2.226-5.553-5.88V11.47h3.124v8.448c0 2.14 1.243 3.31 3.24 3.31 2.197 0 3.672-1.341 3.672-4.11zm-88.114 2.467c0-.972-.8-1.76-1.786-1.76-.987 0-1.786.788-1.786 1.76s.799 1.76 1.786 1.76c.986 0 1.786-.788 1.786-1.76zm-36.726 0c0-.972-.8-1.76-1.786-1.76-.987 0-1.786.788-1.786 1.76s.799 1.76 1.786 1.76c.986 0 1.786-.788 1.786-1.76z"
        fill="#0C6EEF"
        fill-rule="nonzero"
      />
    </symbol>
    <symbol
      class="ThumbDownSymbol"
      id="ThumbDownSymbol"
      viewBox="0 0 18 18"
      xlink="http://www.w3.org/1999/xlink"
      xmlns="http://www.w3.org/2000/svg"
    >
      <g
        fill="none"
        fill-rule="evenodd"
        stroke="none"
        stroke-width="1"
      >
        <g
          transform="translate(-271.000000, -672.000000)"
        >
          <g
            transform="translate(269.000000, 519.000000)"
          >
            <g
              transform="translate(2.000000, 150.000000)"
            >
              <g
                transform="translate(0.000000, 3.000000)"
              >
                <polygon
                  points="0 0 17.0744526 0 17.0744526 18 0 18"
                />
                <path
                  d="M10.6715328,2.25 L4.26861314,2.25 C3.67812165,2.25 3.17300243,2.625 2.95957178,3.165 L0.811036496,8.4525 C0.747007299,8.625 0.711435523,8.805 0.711435523,9 L0.711435523,10.5 C0.711435523,11.325 1.35172749,12 2.13430657,12 L6.62346472,12 L5.94760097,15.4275 L5.92625791,15.6675 C5.92625791,15.975 6.04720195,16.26 6.23928954,16.4625 L6.99341119,17.25 L11.6817713,12.3075 C11.9378881,12.0375 12.0944039,11.6625 12.0944039,11.25 L12.0944039,3.75 C12.0944039,2.925 11.4541119,2.25 10.6715328,2.25 Z M13.5172749,2.25 L13.5172749,11.25 L16.363017,11.25 L16.363017,2.25 L13.5172749,2.25 Z"
                  fill="#5B7186"
                />
              </g>
            </g>
          </g>
        </g>
      </g>
    </symbol>
    <symbol
      class="ThumbUpSymbol"
      id="ThumbUpSymbol"
      viewBox="0 0 18 18"
      xlink="http://www.w3.org/1999/xlink"
      xmlns="http://www.w3.org/2000/svg"
    >
      <g
        fill="none"
        fill-rule="evenodd"
        stroke="none"
        stroke-width="1"
      >
        <g
          transform="translate(-271.000000, -608.000000)"
        >
          <g
            transform="translate(269.000000, 519.000000)"
          >
            <g
              transform="translate(2.000000, 86.000000)"
            >
              <g
                transform="translate(0.000000, 3.000000)"
              >
                <path
                  d="M0,0 L19.9201946,0 L19.9201946,18 L0,18 L0,0 Z M0,0 L19.9201946,0 L19.9201946,18 L0,18 L0,0 Z"
                  id="ThumbUpSymbol-Shape"
                />
                <path
                  d="M10.8897064,1.545 L6.29146148,5.7 C5.98435848,5.9775 5.81005677,6.36 5.81005677,6.7575 L5.81005677,14.25 C5.81005677,15.075 6.55706407,15.75 7.47007299,15.75 L14.940146,15.75 C15.6041525,15.75 16.2017583,15.39 16.4673609,14.8425 L19.1731873,9.135 C19.8703942,7.65 18.6668824,6 16.882365,6 L12.1928191,6 L12.9813268,2.565 C13.0643277,2.19 12.9398264,1.8075 12.6410235,1.5375 C12.1513187,1.1025 11.3711111,1.1025 10.8897064,1.545 Z M2.49002433,15.75 C3.40303325,15.75 4.15004055,15.075 4.15004055,14.25 L4.15004055,8.25 C4.15004055,7.425 3.40303325,6.75 2.49002433,6.75 C1.57701541,6.75 0.83000811,7.425 0.83000811,8.25 L0.83000811,14.25 C0.83000811,15.075 1.57701541,15.75 2.49002433,15.75 Z"
                  fill="#16A085"
                />
              </g>
            </g>
          </g>
        </g>
      </g>
    </symbol>
  </svg>
</div>
`;
